libraries/ArduinoOTA/ArduinoOTA.cpp

```diff
@@ -146,26 +146,26 @@ int ArduinoOTAClass::parseInt(){
   uint8_t index;
   char value;
```

Comment:
value should be declared int, because peek() returns an int that can be -1 on error